src/ConfigFactory.Avalonia.Demo/Models/DemoConfig.cs

```diff
@@ -29,41 +32,41 @@ public partial class DemoConfig : ConfigModule<DemoConfig>
         Filter = "Backups:*.bak",
         InstanceBrowserKey = "some-browser-field-key")]
     [property: Config(
-        Header = "Some Browser Field",
-        Description = "Extended (probably redundant) description of Some Field, a text-based configuration",
-        Category = "General",
-        Group = "Common")]
+        Header = "Browser_Field",
+        Description = "Example_Description",
+        Category = "General_Category",
+        Group = "Common_Group")]
     private string _someBrowserField = string.Empty;
 
     [ObservableProperty]
     [property: DropdownConfig(
         RuntimeItemsSourceMethodName = "GetThings")]
     [property: Config(
-        Header = "Some Dropdown Field",
-        Description = "Extended (probably redundant) description of Some Field, a text-based configuration",
-        Category = "General",
-        Group = "Common")]
+        Header = "Dropdown_Field",
+        Description = "Example_Description",
+        Category = "General_Category",
+        Group = "Common_Group")]
     private string _someDropdownField = string.Empty;
 
     [ObservableProperty]
     [property: BrowserConfig(BrowserMode = BrowserMode.OpenFile)]
     [property: Config(
-        Header = "Some Other Field",
-        Description = "Extended (probably redundant) description of Some Field, a text-based configuration",
-        Category = "General",
-        Group = "Less Common")]
+        Header = "Other_Field",
+        Description = "This string is not included in the translations resource",
+        Category = "General_Category",
+        Group = "Uncommon_Group")]
     private string _someOtherField = string.Empty;
 
     [ObservableProperty]
     [property: BrowserConfig(BrowserMode = BrowserMode.OpenFile)]
     [property: Config(
-        Header = "Some Enum Field",
-        Description = "Extended (probably redundant) description of Some Enum Field, a enum-based configuration",
-        Category = "General",
-        Group = "Less Common")]
+        Header = "Enum_Field",
+        Description = "This string is also not included in the translations resource",
+        Category = "General_Category",
+        Group = "Uncommon_Group")]
     private BrowserMode _someEnumField = BrowserMode.SaveFile;
```

Comment:
It is not possible to resolve a property name that contains whitespaces using Reflection.
Because of that, I was forced to use strings with underscores to resolve the related property in the Translations resource.

Reply:
The property name shouldn't be used at all, nor should reflection (except to get the attribute). I'm confused where you would need reflection for that.

Reply:
Also the original extended descriptions should be restored.

Reply:
Agreed, in this case using Reflection was unnecesary. I got rid of the Reflection in the latest commit.

PS: I remember that my initial idea was to use generic for any Resource class (e.g. Translations.Designer.cs).
But those classes don't have a common interface and decided to use the Reflection.
Later I gave up on idea of using generics but for some reason left the Reflection.

Anyway, I think the latest solution is pretty straight forward.